pkg/terraform/executor.go

```diff
@@ -534,10 +534,8 @@ func (ex *Executor) checkVersion() error {
 		return fmt.Errorf("checking Terraform version: %w", err)
 	}
 
-	constraints, err := version.NewConstraint(requiredVersion)
-	if err != nil {
-		return fmt.Errorf("checking Terraform version: %w", err)
-	}
+	// requiredVersion is const, so we test it in unit tests.
+	constraints, _ := version.NewConstraint(requiredVersion)
```

Comment:
we can log it in debug mode? So we don't lose track of it altogether.

Reply:
Why? It is covered by unit tests, running exactly the same code.

Reply:
It's best to panic in such case, if this error should be found during development itself.